src/main/java/com/prgms/vouchermanager/contorller/customer/CustomerController.java

```diff
@@ -0,0 +1,111 @@
+package com.prgms.vouchermanager.contorller.customer;
+
+import com.prgms.vouchermanager.domain.customer.Customer;
+import com.prgms.vouchermanager.domain.voucher.Voucher;
+import com.prgms.vouchermanager.dto.CreateCustomerDto;
+import com.prgms.vouchermanager.dto.UpdateCustomerDto;
+import com.prgms.vouchermanager.service.customer.CustomerService;
+import com.prgms.vouchermanager.util.io.ConsoleInput;
+import com.prgms.vouchermanager.util.io.ConsoleOutput;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Controller;
+
+import java.util.List;
+
+@Controller
+@Slf4j
+public class CustomerController {
+
+    private final CustomerService customerService;
+    private final ConsoleInput consoleInput;
+
+    private final ConsoleOutput consoleOutput;
+    public CustomerController(CustomerService customerService, ConsoleInput consoleInput, ConsoleOutput consoleOutput) {
+        this.customerService = customerService;
+        this.consoleInput = consoleInput;
+        this.consoleOutput = consoleOutput;
+    }
+
+    public void create() {
+        Customer customer = null;
+        try {
+
+            consoleOutput.printCustomerName();
+            String name = consoleInput.inputCustomerName();
+            consoleOutput.printCustomerEmail();
+            String email = consoleInput.inputCustomerEmail();
+            consoleOutput.printBlackList();
+            int blackList = consoleInput.inputBlackList();
+            customer = customerService.create(new CreateCustomerDto(name,email, blackList));
+            log.info(customer.toString());
+
+        } catch (RuntimeException e) {
+            System.out.println(e.getMessage());
+            log.warn(e.getMessage());
+        }
+
+    }
+
+    public void update() {
+        try {
+            consoleOutput.printCustomerId();
+            Long id = consoleInput.inputCustomerId();
+            consoleOutput.printCustomerName();
+            String name = consoleInput.inputCustomerName();
+            consoleOutput.printCustomerEmail();
+            String email = consoleInput.inputCustomerEmail();
+            consoleOutput.printBlackList();
+            int blackList = consoleInput.inputBlackList();
+            customerService.update(id,new UpdateCustomerDto(name,email,blackList));
+            consoleOutput.printSuccessUpdate();
+
+        } catch (RuntimeException e) {
+            log.warn(e.getMessage());
+        }
+    }
+
+    public void getList() {
+
+        List<Customer> customers = customerService.findAll();
+
+        customers.forEach(customer -> consoleOutput.printCustomer(customer));
```

Comment:
람다식을 메서드 참조로 바꾸시면 좀 더 가독성이 올라갈 것 같습니다.

```suggestion
        customers.forEach(consoleOutput::printCustomer);

```